app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -8,6 +8,7 @@
 @import 'shared_accesses';
 @import 'presentation';
 @import 'admin_panel';
+@import '//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.2/semantic.min.css';
```

Comment:
This is fine for now - but we definitely shouldn't import the entire css file for semantic

Reply:
I believe it's one of these, but I can't seem to find which one exactly contains the css for Pagination component

https://cdnjs.com/libraries/semantic-ui/2.2.2